apps/studio/src/components/PageEditor/ComponentSelector.tsx

```diff
@@ -104,6 +105,14 @@ function ComponentSelector() {
     setSavedPageState,
     setPreviewPageState,
   } = useEditorDrawerContext()
+  const { mutate } = trpc.page.updatePageBlob.useMutation()
+  // TODO: get this dynamically
+  const pageId = 1
+  const siteId = 1
+  const [page] = trpc.page.readPageAndBlob.useSuspenseQuery({
+    pageId,
+    siteId
+  })
   const onProceed = (sectionType: SectionType) => {
     // TODO: add new section to page/editor state
```

Comment:
Sorry didn't understand the TODO here, can explain?

Reply:
oudated, removed